plugins/inlinetoolbar/plugin.js

```diff
@@ -24,7 +24,12 @@
 		} );
 		CKEDITOR.ui.balloonPanel.call( this, editor, definition );
 		/**
-		 * The editor for this balloon panel.
+		 * Registred menu items in inlineToolbarView
+		 * @private
+		 */
+		this.menuItems = [];
```

Comment:
Again I have asked to fix this in https://github.com/ckeditor/ckeditor-dev/pull/992#discussion_r142140769